draft-ietf-dnsop-svcb-https.md

```diff
@@ -651,12 +648,10 @@ See {{incomplete-response}} for possible optimizations of this procedure.
 ## General requirements
 
 Recursive resolvers MUST be able to convey SVCB records with unrecognized
-SvcParamKeys, and MAY treat the entire
-SvcParams portion of the record as opaque.  No part of this specification requires
-recursive resolvers to alter their behavior based on its contents, even if the contents
-are invalid.  Recursive resolvers MAY validate the values of recognized
-SvcParamKeys and reject records containing values 
-which are invalid according to the SvcParam specification.
+SvcParamKeys, and MAY treat the entire SvcParams portion of the record as
+opaque, even if the contents are invalid.  Alternatively, recursive
+resolvers MAY report SERVFAIL to avoid returning a SvcParamValue that is
```

Comment:
@Habbie @vcunat This text change is based on the GENART review from Dale Worley.  Does this still work for your filtering requirements?

Reply:
Filtering?  Is there some context I've missed or forgot?  If it's filtering in sense of Response Policy Zones and similar, I don't see why consider it in this RFC.  That certainly isn't a case of "invalid according to the SvcParam's specification", and SERVFAIL isn't the usual filtering action (because caching).

Regardless of filtering, I wonder if SERVFAIL is the best thing to do in case the wrong record would be only non-mandatory (in the additional section).

Reply:
This is in reference to [this discussion](https://mailarchive.ietf.org/arch/msg/dnsop/DZKcP6tViymFl6axketzstO10i0/) from DNSOP.  I pinged you here because you commented on PR #313, which generated this text, and we want to make sure not regress on consensus.

Reply:
Ah, thanks... I confused the "filtering" meaning.  I think this new text is better.

Some implementations may like to check records to larger extent, some may not, but in any case extensibility needs to be kept in mind.

Reply:
I've relaxed the recommendation to use SERVFAIL specifically.